example/bazel.repos

```diff
@@ -6,64 +6,64 @@ repositories:
   ign_common:
     type: git
     url: https://github.com/ignitionrobotics/ign-common
-    version: bazel
+    version: bazel-common3
   ign_fuel_tools:
     type: git
     url: https://github.com/ignitionrobotics/ign-fuel-tools
-    version: bazel
+    version: bazel-fuel-tools5
   ign_gazebo:
     type: git
     url: https://github.com/ignitionrobotics/ign-gazebo
-    version: bazel
+    version: bazel-gazebo4
   ign_gui:
     type: git
     url: https://github.com/ignitionrobotics/ign-gui
-    version: bazel
+    version: bazel-gui4
   ign_launch:
     type: git
     url: https://github.com/ignitionrobotics/ign-launch
-    version: bazel
+    version: bazel-launch3
   ign_math:
     type: git
     url: https://github.com/ignitionrobotics/ign-math
-    version: bazel
+    version: bazel-math6
   ign_msgs:
     type: git
     url: https://github.com/ignitionrobotics/ign-msgs
-    version: bazel
+    version: bazel-msgs6
   ign_physics:
     type: git
     url: https://github.com/ignitionrobotics/ign-physics
-    version: bazel
+    version: bazel-physics3
   ign_plugin:
     type: git
     url: https://github.com/ignitionrobotics/ign-plugin
-    version: bazel
+    version: bazel-plugin1
   ign_rendering:
     type: git
     url: https://github.com/ignitionrobotics/ign-rendering
-    version: bazel
+    version: bazel-rendering4
   ign_sensors:
     type: git
     url: https://github.com/ignitionrobotics/ign-sensors
-    version: bazel
+    version: bazel-sensors4
   ign_tools:
     type: git
     url: https://github.com/ignitionrobotics/ign-tools
-    version: bazel
+    version: bazel-tools1
   ign_transport:
     type: git
     url: https://github.com/ignitionrobotics/ign-transport
-    version: bazel
+    version: bazel-transport9
   dart:
     type: git
     url: https://github.com/azeey/dart
-    version: bazel
+    version: bazel-azeey
   ogre2:
     type: git
     url: https://github.com/osrf/ogre-2.1-release
     version: bazel
```

Comment:
same here.

Reply:
And here: https://github.com/ignition-forks/ogre-2.1-release